README.md

```diff
@@ -100,14 +102,21 @@ Essentially, clone the repo, and run `python vid2cleantxt/transcribe.py --input-
     -   add `--depth=1` to above to clone just the current code & objects and **will be faster**
 2.  `cd vid2cleantxt/`
 3.  `pip install -r requirements.txt`
-4.  `python vid2cleantxt/transcribe.py --input-dir "example_JFK_speech/TEST_singlefile"`
+4.  `python vid2cleantxt/transcribe.py --input-dir "examples/TEST_singlefile"`
     -   in this example\*, all video and audio files in the repo example "example_JFK_speech/TEST_singlefile" would be transcribed.
+    -   download the video with `python examples/TEST_singlefile/dl_src_video.py`
 
 -   if you are new, you can clone with [github desktop](https://desktop.github.com/)
 -   if neither option works for you, check out the Colab notebooks distributed with this repo.
 
 > \* the example videos need to be downloaded with the scripts in the relevant dirs, such as `python examples/TEST_singlefile/dl_src_video.py`
 
+## Use as package
+1.  `pip install .`
+2.  `spacy download en_core_web_sm`
```

Comment:
does the standard way of getting the package not work? do you have reproducible code? going to be kind of awkward to get this on pypi and force downloading the package manually.

I'd like to at least try it and see it not work before changing the way it's downloaded